### DISCUSSION TITLE
Person Disambiguation of Creators

### DISCUSSION THREAD

**Discussion by ghost** [wgu-opensource/osmt#72] [General]
This question has come up in other schemas and federated content systems we've been involved with.

The schemas has a place for "contributor" (https://purl.org/ctdlasn/terms/creator) but no formal definition of who/what a contributor is.

How are contributors disambiguated? For instance, GRCSchema.org has a schema for a Contributor (https://grcschema.org/Contributor) that ties the contributor to either a person or an organization (DISA and others like to have the contributor as the whole organization).

This allows for disambiguation of contributors so that we know "Joe Schlabotnik" is this specific Joe Schlabotnik versus some other Joe Schlabotnik.

How is the OSMT handling disambiguation of contributors?

**Comment by arblackwell1** [comment]
@ottonomy Can you help answer Dorian's question here?
